src/Form/Type/FormatterType.php

```diff
@@ -58,18 +59,25 @@
      */
     private $templateManager;
 
+    /**
+     * @var ToolbarManagerInterface
+     */
+    private $toolbarManager;
+
     public function __construct(
         Pool $pool,
         TranslatorInterface $translator,
         ConfigManagerInterface $configManager,
         ?PluginManagerInterface $pluginManager = null,
-        ?TemplateManagerInterface $templateManager = null
+        ?TemplateManagerInterface $templateManager = null,
```

Comment:
are we sure this should stay nullable on master?

Reply:
Maybe not, but this is not the PR to change it, don't do [evil merges](https://blog.joolsadams.org/2013/02/15/evil-merges-in-git/)

Reply:
I know I know, I just now saw it and was wondering :smile: